llvm/lib/Target/RISCV/RISCVExpandPseudoInsts.cpp

```diff
@@ -169,7 +169,7 @@ bool RISCVExpandPseudo::expandCCOp(MachineBasicBlock &MBB,
   CC = RISCVCC::getOppositeBranchCondition(CC);
 
   // Insert branch instruction.
-  BuildMI(MBB, MBBI, DL, TII->getBrCond(CC))
+  BuildMI(MBB, MBBI, DL, TII->getBrCond(CC, false))
```

Comment:
Maybe you can use a default argument to avoid changing all call sites of `getBrCond`?

Reply:
done